TG-Netflix/src/pages/account/AccountPage.styled.jsx

```diff
@@ -0,0 +1,429 @@
+flex-direction: column;
+align-items: flex-start;
+
+.member-date {
+  margin-left: 0;
+}
+}
+`;
+
+export const AccountSection = styled.div`
+border-top: 1px solid #999;
+display: flex;
+justify-content: space-between;
+font-size: 1.58rem;
+padding-bottom: 1.2rem;
+min-height: 8rem;
+
+h2 {
+  color: #737373;
+  font-size: 1.8rem;
+  font-weight: 500;
+}
+
+p {
+  margin-bottom: 0.8rem;
+}
+
+p:first-child {
+  font-weight: bold;
+}
+
+.account-section-header {
+  padding-top: 1.2rem;
+  width: 23%;
+  font-weight: 500;
+  color: #737373;
+}
+
+.subsections-container {
+  display: flex;
+  flex-direction: column;
+  width: 74%;
+}
+
+.account-subsection {
+  padding-top: 1.2rem;
+
+  ul {
+    list-style: none;
+  }
+}
+
+.arrow-mobile {
+  display: none;
+}
+
+.membership-section-divider {
+  opacity: 0.2;
+}
+
+.payment-method {
+  display: flex;
+  font-weight: bold;
+  width: 8rem;
+  justify-content: space-between;
+
+  img {
+    width: 2.2rem;
+    height: 2.5rem;
+  }
+}
+
+.cancel-btn {
+  width: 19.5rem;
+  height: 3.8rem;
+  padding: 1rem;
+  font-size: 1.3rem;
+  font-weight: 500;
+  border: none;
+  background-color: #e6e6e6;
+  background-image: linear-gradient(180deg, #e6e6e6, #ddd);
+  box-shadow: 0 1px 0 rgb(0 0 0 / 20%);
+  margin-top: 0.8rem;
+  cursor: pointer;
+}
+
+.cancel-btn:hover {
+  background: #eaeaea;
+}
+
+.cancel-btn-mobile {
+  display: none;
+  width: 95%;
+  margin: 1rem auto;
+  height: 5rem;
+  padding: 1rem;
+  font-size: 1.8rem;
+  font-weight: 500;
+  border: none;
+  background-color: #e6e6e6;
+  background-image: linear-gradient(180deg, #e6e6e6, #ddd);
+  box-shadow: 0 1px 0 rgb(0 0 0 / 20%);
+  margin-top: 0.8rem;
+  cursor: pointer;
+}
+
+.cancel-btn-mobile:hover {
+  background: #eaeaea;
+}
+
+.settings-section {
+  width: 74%;
+  align-self: flex-start;
+
+  li {
+    text-align: left;
+    padding-bottom: 1rem;
+  }
+}
+
+@media screen and (max-width: 820px) {
+  .subsections-container {
+    width: 60%;
+  }
+
+  .settings-section {
+    width: 60%;
+  }
+}
+
+@media screen and (max-width: 740px) {
+    flex-direction: column;
+
+    .account-section-header {
+      width: 100%;
+    }
+
+    .subsections-container {
+      width: 100%;
+    }
+}
+
+@media screen and (max-width: 500px) {
+background-color: #FFF;
+border: 1px solid #999;
+flex-direction: column;
+padding: 1rem;
+margin-bottom: 0.7rem;
+
+a {
+  display: flex;
+  justify-content: space-between;
+}
+
+.arrow-mobile {
+ color: #999;
+ height: 2.5rem;
+ display: flex;
+ align-items: center;
+}
+
+.account-section-header {
+  width: 100%;
+    }
+  
+    .subsections-container {
+      width: 100%;
+    }
+
+.membership-section-divider {
+  opacity: 0.8;
+}
+
+.cancel-btn {
+  display: none;
+}
+
+.cancel-btn-mobile {
+  display: block;
+}
+
+.settings-section {
+  width: 100%;
+
+  li {
+      text-align: left;
+        padding-top: 1.5rem;
+        padding-bottom: 1.5rem;
+         border-top: 1px solid #ccc;
+  }
+}
```

Comment:
vs code verwacht hier een curly bracket?  misschien die geopend is op line 235, er staat een closing bracket op line 301, als je deze verplaatst naar line 284 gaat de error weg.